### ISSUE TITLE
Draft role descriptions for initial team

### ISSUE THREAD

**Issue by alexpandel** [18F/public-benefits-studio#3]
**Why:** so we can hire a team!

**Prerequisite:** #4 

Note: [the 10x materials/process](https://docs.google.com/document/d/1Qj35ard4yww_86Ikw23FhIUgEGyiECDb074ZrFj3uDU/edit) look like a great model to emulate!

**Definition of done:**
- [x] we've got a description of each of the roles drafted (modifying [these versions 🔒](https://docs.google.com/document/d/15P_mUupnm-11F1b_a6xzs4Sw-PmWomq8huM1TPLC98E/edit#heading=h.qqqxl291ufh) as follows):
  - strip out any details that don't apply to the Studio
  - make sure the hard skills reflect what the Studio team needs (and change to reflect that we're looking for a breadth of skills _across the individuals_ in each similar role, rather than trying to draw boundaries around particular specialties like 10x did)
  - include the soft skills we'd like to see in each team member
- [x] we've got those drafts in front of the right folks (at a minimum, the rest of the facilitation team, incl Ashley, but prob a wider circle too?)

**Left to do:**
- [x] PM
- [x] Design
- [x] Engineering
- [ ] Deputy

**Comment by alexpandel** [comment]
calling this done for now; spinning up separate card for Deputy role
